### PR TITLE
[devtools-translate] Russian - What's New in DevTools (Chrome 97)

### LINKED ISSUE #1783: [devtools-translate] Russian - What's New in DevTools (Chrome 97)
Post: https://developer.chrome.com/blog/new-in-devtools-97/

Translator: @solarrust
Reviewers: @webmaxru, @kateryna-prokopenko

Translation ready by: Dec 20

### PR DESCRIPTION
**Pull request by solarrust**
Fixes #1783 

Hi! 

@kateryna-prokopenko @webmaxru notes for you:
- Recorder panel don't have Russian translation right now in DevTools. So I didn't translate this name for now.
- Is «автокомплит» fairly self-explanatory term already? Or do we still have to use «автодополнение»?
- I use «панели инструментов устройства» as translation for Device Mode. This phrase is used in DevTools right now. May be you have more elegant shape =)

### CODE REVIEW

File: site/ru/blog/new-in-devtools-97/index.md [hunk 1]
```diff
@@ -30,70 +29,101 @@ draft: true
...
+Ваши настройки DevTools теперь синхронизируются между устройствами когда вы включаете синхронизацию
+профиля Chrome. Вы можете изменить настройки синхронизации зайдя в **Настройки** (Settings) >
+**Синхронизация** (Sync)  > **Включить синхронизацию настроек** (Enable settings sync).
 
-{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/LUwFNTDyP22L1euSGg73.png", alt="DevTools sync settings", width="800", height="654" %}
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/LUwFNTDyP22L1euSGg73.png", alt="Настройки
+синхронизации DevTools", width="800", height="654" %}
 
 <!-- This new setting makes it easier for you to work across devices. For example, the following appearance settings are synced so you have a consistent experience across devices and don’t need to re-define the same settings again. Learn more about the sync feature in [DevTools customization](/docs/devtools/customize/). -->
+Эта настройка облегчает работу на разных устройствах. Например, следующие настройки внешнего вида
```
Comment (webmaxru) [inline site/ru/blog/new-in-devtools-97/index.md:116]:
```suggestion
Эта функция облегчает работу на разных устройствах. Например, следующие настройки внешнего вида
```

File: site/ru/blog/new-in-devtools-97/index.md [hunk 2]
```diff
@@ -30,70 +29,101 @@ draft: true
...
+синхронизированы и ваш пользовательский опыт на разных устройствах будет консистентным, вам не нужно
```
Comment (webmaxru) [inline site/ru/blog/new-in-devtools-97/index.md:117]:
```suggestion
могут быть синхронизированы, так что ваш пользовательский опыт на разных устройствах будет консистентным без необходимости
```

File: site/ru/blog/new-in-devtools-97/index.md [hunk 3]
```diff
@@ -30,70 +29,101 @@ draft: true
...
+заново настраивать те же настройки.
```
Comment (webmaxru) [inline site/ru/blog/new-in-devtools-97/index.md:118]:
```suggestion
настраивать эти параметры заново.
```

File: site/ru/blog/new-in-devtools-97/index.md [hunk 4]
```diff
@@ -30,70 +29,101 @@ draft: true
...
 
-{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/t8SQuZ4mE2xiLVxaZz11.png", alt="appearance settings", width="800", height="584" %}
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/t8SQuZ4mE2xiLVxaZz11.png", alt="Настройки внешнего вида", width="800", height="584" %}
 
 <!-- This feature is experimental at the moment, the team is still actively working on it. If you have any feedback, please share with us [here](https://crbug.com/1245541). -->
+В данный момент это экспериментальная функция, наша команда все ещё активно работает над ней.
+Пожалуйста, поделитесь с нами любым фидбэком [здесь](https://crbug.com/1245541).
```
Comment (webmaxru) [inline site/ru/blog/new-in-devtools-97/index.md:124]:
```suggestion
Ждем ваши отзывы [здесь](https://crbug.com/1245541).
```

**Review by webmaxru — CHANGES_REQUESTED** [review]

File: site/ru/blog/new-in-devtools-97/index.md [hunk 5]
```diff
@@ -30,70 +29,101 @@ draft: true
...
+пользовательские сценарии.
 
 <!-- [Open the **Recorder** panel](/docs/devtools/recorder/#open). Follow the instructions on screen to start a new recording.  -->
+[Откройте панель **Recorder**](/docs/devtools/recorder/#open). Следуйте инструкциям на
+экране чтобы начать новую запись.
 
 <!-- For example, you can record the coffee checkout process with this [coffee ordering demo](https://coffee-cart.netlify.app/) application. After adding a coffee and filling out payment details, you can end the recording, replay the process or click on the **Measure performance** button to measure the user flow in the **Performance** panel. -->
+Например, вы можете записать процесс заказа кофе с помощью этого [демо приложения по заказу
+кофе](https://coffee-cart.netlify.app/). После выбора кофе и ввода платёжных данных вы можете
+закончить запись, проиграть её с начала или кликнуть на кнопку **Measure performance** для просмотра пользовательских данных в панели **Производительность** (Performance).
```
[the remark this answers is not in the document]
Comment (kateryna-prokopenko) [reply site/ru/blog/new-in-devtools-97/index.md:45]:
```suggestion
закончить запись, воспроизвести её сначала или кликнуть на кнопку **Measure performance** для просмотра пользовательских данных во вкладке **Производительность** (Performance).
```

File: site/ru/blog/new-in-devtools-97/index.md [hunk 6]
```diff
@@ -30,70 +29,101 @@ draft: true
...
 
 <!-- Go to the **Recorder** panel [documentation](/docs/devtools/recorder/) to learn more with the step-by-step tutorial! -->
+Обратитесь к [документации](/docs/devtools/recorder/) по панели **Recorder** чтобы узнать больше с пошаговыми туториалами.
```
[the remark this answers is not in the document]
Comment (kateryna-prokopenko) [reply site/ru/blog/new-in-devtools-97/index.md:48]:
```suggestion
Обратитесь к [документации](/docs/devtools/recorder/) по вкладке **Recorder**, чтобы узнать больше из пошагового руководства.
```

File: site/ru/blog/new-in-devtools-97/index.md [hunk 7]
```diff
@@ -30,70 +29,101 @@ draft: true
...
 
 <!-- The **Recorder** panel is a preview feature. Our team is still actively working on it and we are looking for your [feedback](https://goo.gle/recorder-feedback) for further enhancements. -->
+Функция **Recorder** находится в раннем доступе. Наша команда все ещё активно работает над ней, и мы ждем [ваши отзывы](https://goo.gle/recorder-feedback) для дальнейших улучшений.
```
Comment (kateryna-prokopenko) [inline site/ru/blog/new-in-devtools-97/index.md:51]:
```suggestion
Вкладка **Recorder** находится в раннем доступе. Наша команда все ещё активно работает над ней, и мы ждем [ваши отзывы](https://goo.gle/recorder-feedback) для дальнейших улучшений.
```

File: site/ru/blog/new-in-devtools-97/index.md [hunk 8]
```diff
@@ -30,70 +29,101 @@ draft: true
...
 
-{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/3EpVa15PtbhFwwszqyWF.png", alt="Recorder panel", width="800", height="540" %}
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/3EpVa15PtbhFwwszqyWF.png", alt="Панель Recorder", width="800", height="540" %}
```
Comment (kateryna-prokopenko) [inline site/ru/blog/new-in-devtools-97/index.md:53]:
```suggestion
{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/3EpVa15PtbhFwwszqyWF.png", alt="Вкладка Recorder", width="800", height="540" %}
```

File: site/ru/blog/new-in-devtools-97/index.md [hunk 9]
```diff
@@ -30,70 +29,101 @@ draft: true
...
 
-Chromium issue: [1215072](https://crbug.com/1215072)
+Задача в трекере Chromium: [1215072](https://crbug.com/1215072)
 
 
 <!-- ## Improved code debugging experience {: #debugging } -->
+## Улучшен процесс отладки кода {: #debugging }
 
 <!-- Column numbers are now included in the output error in the Console. Having easy access to the column number is essential for debugging especially with minified JavaScript. -->
+Номера колонок добавлены в вывод ошибки в Консоли (Console). Простой доступ к номеру колонки
```
[the remark this answers is not in the document]
Comment (kateryna-prokopenko) [reply site/ru/blog/new-in-devtools-97/index.md:94]:
```suggestion
В вывод ошибки в Консоли (Console) теперь добавлены номера колонок. Простой доступ к номеру колонки
```

File: site/ru/blog/new-in-devtools-97/index.md [hunk 10]
```diff
@@ -30,70 +29,101 @@ draft: true
...
 
-Chromium issue: [1073064](https://crbug.com/1073064)
+Задача в трекере Chromium: [1073064](https://crbug.com/1073064)
 
 
 <!-- ## [Experimental] Syncing DevTools settings across devices {: #sync } -->
+## [Эксперимент] Синхронизация настроек DevTools между устройствами {: #sync }
 
 <!-- Your DevTools settings are now synced across devices by default when you turn on Chrome profile sync. You can change the DevTools sync settings via **Settings** > **Sync** > **Enable settings sync**.  -->
+Ваши настройки DevTools теперь синхронизируются между устройствами когда вы включаете синхронизацию
```
[the remark this answers is not in the document]
Comment (kateryna-prokopenko) [reply site/ru/blog/new-in-devtools-97/index.md:108]:
```suggestion
Ваши настройки DevTools теперь синхронизируются между устройствами, когда вы включаете синхронизацию
```

**Comment by kateryna-prokopenko** [comment]
Hi Alena!
I guess it's fine not to translate the Recorder panel name for now.
Автодополнение still sounds better.
It's best to use the names that are used in DevTools even if these don't sound super elegant:)
Thanks!

**Comment by solarrust** [comment]
@webmaxru @kateryna-prokopenko apply all your suggestions, thank you! 🙏

File: site/_includes/partials/devtools/ru/whats-new.md [hunk 11]
```diff
@@ -10,7 +10,16 @@
 * [Improved code debugging experience](/ru/blog/new-in-devtools-97/#debugging)
 * [Syncing DevTools settings across devices](/ru/blog/new-in-devtools-97/#sync) -->
 
+### Chrome 97 {: #chrome97 }
+
+* [Ранний доступ: новая панель Recorder](/ru/blog/new-in-devtools-97/#recorder)
```
Comment (kateryna-prokopenko) [inline site/_includes/partials/devtools/ru/whats-new.md:15]:
```suggestion
* [Ранний доступ: новая вкладка Recorder](/ru/blog/new-in-devtools-97/#recorder)
```

File: site/ru/blog/new-in-devtools-97/index.md [hunk 12]
```diff
@@ -30,70 +29,101 @@ draft: true
 
 
 <!-- ## Preview feature: New Recorder panel {: #recorder } -->
+## Ранний доступ: новая панель Recorder  {: #recorder }
```
Comment (kateryna-prokopenko) [inline site/ru/blog/new-in-devtools-97/index.md:32]:
```suggestion
## Ранний доступ: новая вкладка Recorder  {: #recorder }
```

File: site/ru/blog/new-in-devtools-97/index.md [hunk 13]
```diff
@@ -30,70 +29,101 @@ draft: true
...
 
 <!-- Use the new **Recorder** panel to record, replay and measure user flows.  -->
+Используйте новую панель **Recorder**, чтобы записать, воспроизвести или измерить
```
Comment (kateryna-prokopenko) [inline site/ru/blog/new-in-devtools-97/index.md:35]:
```suggestion
Используйте новую вкладку **Recorder**, чтобы записать, воспроизвести или измерить
```

File: site/ru/blog/new-in-devtools-97/index.md [hunk 14]
```diff
@@ -30,70 +29,101 @@ draft: true
...
+пользовательские сценарии.
 
 <!-- [Open the **Recorder** panel](/docs/devtools/recorder/#open). Follow the instructions on screen to start a new recording.  -->
+[Откройте вкладку **Recorder**](/docs/devtools/recorder/#open). Следуйте инструкциям на
+экране, чтобы начать новую запись.
 
 <!-- For example, you can record the coffee checkout process with this [coffee ordering demo](https://coffee-cart.netlify.app/) application. After adding a coffee and filling out payment details, you can end the recording, replay the process or click on the **Measure performance** button to measure the user flow in the **Performance** panel. -->
+Например, вы можете записать процесс заказа кофе с помощью этого [демо приложения
+](https://coffee-cart.netlify.app/). После выбора кофе и ввода платёжных данных вы можете
+закончить запись, воспроизвести её сначала или кликнуть на кнопку **Measure performance** для просмотра пользовательских данных в панели **Производительность** (Performance).
```
Comment (kateryna-prokopenko) [inline site/ru/blog/new-in-devtools-97/index.md:45]:
```suggestion
закончить запись, воспроизвести её сначала или кликнуть на кнопку **Measure performance** для просмотра пользовательских данных во вкладке **Производительность** (Performance).
```

File: site/ru/blog/new-in-devtools-97/index.md [hunk 15]
```diff
@@ -30,70 +29,101 @@ draft: true
...
 
 <!-- Go to the **Recorder** panel [documentation](/docs/devtools/recorder/) to learn more with the step-by-step tutorial! -->
+Обратитесь к [документации](/docs/devtools/recorder/) по панели **Recorder**, чтобы узнать больше из пошагового руководства.
```
Comment (kateryna-prokopenko) [inline site/ru/blog/new-in-devtools-97/index.md:48]:
```suggestion
Обратитесь к [документации](/docs/devtools/recorder/) по вкладке **Recorder**, чтобы узнать больше из пошагового руководства.
```

**Review by kateryna-prokopenko — COMMENTED** [review]
Sorry, several more comments. In general, panel/tab should be translated as вкладка and only pane as панель, thanks!

**Comment by solarrust** [comment]
@kateryna-prokopenko done, thank you!

**Review by kateryna-prokopenko — APPROVED** [review]

**Pull request merged**